src/main/java/org/openjdk/jextract/impl/Logger.java

```diff
@@ -93,6 +87,33 @@ public void info(Position pos, String key, Object... args) {
         errWriter.println(msg);
     }
 
+    public void clangErr(Position pos, String msg) {
```

Comment:
Optional: maybe this is just a single method that takes a clang diagnostic, and renders it appropriately?

Reply:
No clang API binding related stuff in Logger so far. Better to leave it at that I think.